src/Components/ServiceScene/LogsVolume/LogsVolumeContainerScene.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+
+import { css } from '@emotion/css';
+
+import { GrafanaTheme2 } from '@grafana/data';
+import {
+  SceneComponentProps,
+  SceneFlexItem,
+  SceneFlexLayout,
+  SceneObjectBase,
+  SceneObjectState,
+} from '@grafana/scenes';
+import { useStyles2 } from '@grafana/ui';
+
+import { InsightsWidgetScene } from './InsightsWidgetScene';
+import { LogsVolumePanel } from './LogsVolumePanel';
+
+export interface LogsVolumeContainerSceneState extends SceneObjectState {
+  body?: SceneFlexLayout;
+}
+
+export class LogsVolumeContainerScene extends SceneObjectBase<LogsVolumeContainerSceneState> {
+  constructor(state: LogsVolumeContainerSceneState) {
+    super(state);
+    this.addActivationHandler(this.onActivate.bind(this));
+  }
+
+  private onActivate() {
+    this.setState({
+      body: new SceneFlexLayout({
+        direction: 'column',
+        children: [
+          new SceneFlexItem({
+            body: new LogsVolumePanel({}),
```

Comment:
Super minor, but if we could build these without passing an empty object, :cheffkiss: